recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
 
 build:
   number: 1
-  skip: true  # [osx]
+  skip: true  # [osx or (linux and s390x)]
 
 requirements:
   run:
```

Comment:
I guess `pkg_name x.y.z` is fine because at runtime it will be resolved as `pkg_name x.y.z.*`

see https://staging.continuum.io/package/prefect/fs/cuda-libraries-static-feedstock/pr1/e99a810/linux-64/cuda-libraries-static-12.4.1-h06a4308_1.conda

Reply:
I think the precision of the pinnings matches their versions anyway - i.e. the * isn't necessary anyway. (only checked a couple.)